index.html

```diff
@@ -0,0 +1,275 @@
+
+        /* Please comment out this line to see Exercise 2 - Different approach. And the one on the bottom
+
+        var giveMeSpeed = prompt("What is your speed in km/h?")
+        function calculateSpeed(distance) {
+            var result = (distance * 1.60934) / giveMeSpeed
+            return result 
+        }
+        var distanceCluj = 281.5;
+        var distanceIasi = 243.5;
+
+        alert(`A traveler by car will take ${Math.round(calculateSpeed(distanceCluj)*100)/100} hours to travel ${distanceCluj} miles (${Math.round((distanceCluj*1.60934)*100)/100} km) from Bucharest to Cluj, with an average speed of ${giveMeSpeed} km/h.`)
+        alert(`A traveler by car will take ${Math.round(calculateSpeed(distanceIasi)*100)/100} hours to travel ${distanceIasi} miles (${Math.round((distanceIasi*1.60934)*100)/100} km) from Cluj to Iasi, with an average speed of ${giveMeSpeed} km/h.`)
+
+        Please comment out this line to see Exercise 2 - Different approach. */
+
+        /* Exercise 3 - Complete - Reviewed
+
+         Question: Is there an easier way to round to 2 decimals?
+         Answer: Found Math.round
+         !!! TRY WITH parseFloat(integer) and parseINT(integer).
+         With parseFloat does not work with parseINT is the same as Math.round. */
+
+        /* Please comment out this line to see Exercise 3. And the one on the bottom.
+
+        function exchangeUSD_RON (EUR) {
+            var calcRON = EUR * 4.86;
+            // var calcRONtoDecimal = calcRON.toFixed(2); old way I used to round going to leave it on
+            var calcUSD = EUR * 1.16;
+            // var calcUSDtoDecimal = calcUSD.toFixed(2); old way I used to round going to leave it on
+            return alert(`${EUR} EUR is ${parseInt(calcRON)} RON or ${parseInt(calcUSD)} USD.`);
+        }
+        exchangeUSD_RON(prompt("Please insert the sum in EUR:"))
+
+        Please comment out this line to see Exercise 3 */
+
+        // Exercise 4 - Complete - Reviewed but need help in counting people over 18
+        // I can do it in a different way, but I want it with n++.
+
+        /* Please comment out this line to see Exercise 4. And the one on the bottom.
+        
+        function areYouAdult(insertAge) {
+            if (insertAge < 0) {
+                return alert("Liar! Liar! Pants on fire!");
+            } else if (insertAge > 70) {
+                return alert("If you survived this long you can do whatever the fuck you want, smoke some weed, shoot some heroin and enjoy the rest of your days.")
+            } else if (insertAge < 18) {
+                return alert("You're not 18! Can't watch pornache!");
+            }
+             else {
+                return alert("Congrats, you can watch pornache!");
+            }
+        }
+        function callingAges(){
+            i = 1;
+            n = 0;
+            while (i <=4){
+                var ask = areYouAdult(prompt(`How old are you? Person ${i}.`));
```

Comment:
```suggestion
                var ask = prompt(`How old are you? Person ${i}.`);
```
It will work just fine if you remove the `areYouAdult` call